lya/data.py

```diff
@@ -1,43 +1,73 @@
 from .conf import lya_data_path
-
+import numpy as np
+import matplotlib.pyplot as plt
 
 class Data(object):
-    def __init__(self, x, y, y_unc):
-        self.x = x
-        self.y = y
-        self.y_unc = y_unc
 
+    def __init__(self,
+                 k=None,
+                 P_k=None,
+                 P_k_unc=None,
+                 ):
 
-def load_lya_data(filename=lya_data_path):
-    """Load lyman-alpha data
+        self.k = k
+        self.P_k = P_k
+        self.P_k_unc = P_k_unc
 
-    Parameters
-    ----------
-    filename : str
-        Path to ly-a data
+    def load_lya_txt(self, filename=lya_data_path, return_arrays=False):
```

Comment:
If we do make `Data` a generic data class, this and `plot_pk` should probably not be class methods

Reply:
Why not? I guess it depends on what you mean by generic here, but we know `Data` is always going to be Lya data (for this project of course) and will have `P_k` anyway,  so it makes sense to have those methods as instance methods.